docs/UserGuide.md

```diff
@@ -274,8 +366,25 @@ Format: `edit NRIC [n/NAME] [nr/NRIC] [p/PHONE] [e/EMAIL] [a/ADDRESS] [bd/BIRTH_
 Examples:
 
 * `edit S2233556T p/91642345 re/NORTH` Edits the phone number of the person identified by `S2233556T` to be `91642345` and region to be `NORTH`.
-* `edit S8833657U ag/68 r/HIGH` Edits the age of the person identified by `S8833657U` to be `68` and risk level to be `HIGH`. However, if `S8833657U` identifies a volunteer, the risk level edit will be ignored.
+* `edit S8833657U re/CENTRAL r/HIGH` Edits the region of the person identified by `S8833657U` to be `CENTRAL` and risk level to be `HIGH`. However, if `S8833657U` identifies a volunteer, the risk level edit will be ignored.
+
+#### Editing a pair by index: `edit_pair` (COMING SOON)
+
+<div markdown="span" class="alert alert-info">:information_source: **Info:**
+This feature has not been implemented yet.
+</div>
+
+Edits an existing pair based on their index in the pairs list.
+
+Format: `edit_pair INDEX [eic/ELDERLY_NRIC] [vic/VOLUNTEER_NRIC]`
+
+* Edits the pair at the specified `INDEX`. The index refers to the index number shown in the displayed pairs list. The index **must be a positive integer** 1, 2, 3, …​
```

Comment:
```suggestion
* Edits the pair at the specified `INDEX`. 
```
I think can put `INDEX` in the advanced info so no need to repeat it multiple times

Reply:
Resolved in #[4e9d019](https://github.com/AY2223S2-CS2103T-W12-1/tp/pull/181/commits/4e9d019b894de4f64d9750e12ad8f1b1a16fed11)